src/components/ButtonComponent.spec.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import ButtonComponent from './ButtonComponent';
+
+describe('ButtonComponent', () => {
+  it('should render the Button Component', () => {
```

Comment:
We should have tests for different type of buttons/states. This can go in your next PR. Please write an issue and assign it to yourself :)